.prettierrc.yml

```diff
@@ -8,4 +8,20 @@ trailingComma: none
 bracketSpacing: true
 bracketSameLine: true
```

Comment:
Since we're doing a big set of changes already, what's the appetite for changing this to `false`? I find brackets on the same line to make visually determining props and children difficult to do at a glance.

```typescript
<MyComp
  arg1="something"
  arg2="something"
  arg3="something"
  arg4="something"
  arg5="something"
  arg6="something">
  something
</MyComp>
```

Is much harder to read (in my opinion) than...
```typescript
<MyComp
  arg1="something"
  arg2="something"
  arg3="something"
  arg4="something"
  arg5="something"
  arg6="something"
>
  something
</MyComp>
```

Visually scanning, the latter is much easier to immediately see where props end and children begin.

Reply:
I agree to it. I faced this "searching for closing bracket" at times.

Reply:
agreed. let's make this change